README.md

```diff
@@ -2,6 +2,10 @@
 
 # Python Codespace Template
 
+* **Who is this for?** _Educators of all levels_. 
+* **How much experience do students need?** _Zero_. This template is built with basic elements complete with comments so it can be used in beginner to advanced lessons.
+* **Prerequisites:** _None_. This template will provide a working and deployable web app you can immediately extend for your needs.
```

Comment:
This repository doesn't have a web application. 
```suggestion
* **Prerequisites:** _None_. This template will provide a working Jupyter Notebook with Pandas using a dataset so that you can immediately start analyzing data.
```